apps/dashboard/src/components/conditions-editor/value-editor.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo } from 'react';
+import { autocompletion } from '@codemirror/autocomplete';
+import { useValueEditor, ValueEditorProps } from 'react-querybuilder';
+
+import { completions } from '@/utils/liquid-autocomplete';
+import { InputRoot, InputWrapper } from '@/components/primitives/input';
+import { Editor } from '@/components/primitives/editor';
+import { variables } from '@/components/conditions-editor/conditions-editor';
+
+export const ValueEditor = React.memo((props: ValueEditorProps) => {
```

Comment:
This component represents the `value` input field. It renders different fields depending on the operator picked. The fields are CodeMirror `Editor` components.

![Screenshot 2025-01-14 at 12 35 55](https://github.com/user-attachments/assets/a1ca3d22-8381-4866-b8a7-4f6320e0a6d6)